common/test/common/JsonComponentTest.scala

```diff
@@ -55,7 +57,7 @@ class JsonComponentTest extends FlatSpec with Matchers with ExecutionContexts {
 
     val result = Future {
       val request = FakeRequest("GET", "http://foo.bar.com/data.json")
-      JsonComponent(obj("name" -> "foo"))(request).result
+      JsonComponent(obj("name" -> "foo"))(request, play.api.libs.json.Writes.JsValueWrites).result
```

Comment:
Do you still need to specific the `JsValueWrites`? Also you might need `WithRefreshStatus()` here